pkg/services/hcloud/server/server.go

```diff
@@ -56,9 +60,28 @@ const (
 	// Continuous RequeueAfter in BootToRealOS.
 	requeueIntervalBootToRealOS = 10 * time.Second
 
+	// requeueAfterEnableRescueSystem: the hcloud API needs roughly 3 seconds
```

Comment:
do we really wanna define constants for all of this? I believe by now it is easier to just define this in the code.. They are not understandable anymore anyway just from the name